src/app/pages/details/details.hbs

```diff
@@ -0,0 +1,172 @@
+<style scoped>
+    @import url(/app/pages/details/details.css);
+    @import url(https://fonts.googleapis.com/css?family=Lato:400,900);
+</style>
+<section class="floating-menu">
```

Comment:
I did give the appropriateness of the section tag some thought as I applied it. Each section is separately thematic, although the endorsement isn't so much a grouping, so I've demoted it to div. All the others strike me as sections. The floating menu is a thematic grouping, with a .title that functions a heading, while all the others begin with an `h1`.